modules/libffi/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "homepage": "https://github.com/libffi/libffi",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
Are you willing to be a maintainer of this module?

Reply:
If possible, I would prefer not to be an official maintainer, as I'm not involved in ffi at all. I'm only adding it here because I'm currently adding https://github.com/AravisProject/aravis and all its dependencies to bcr.